source/appModules/outlook.py

```diff
@@ -419,6 +437,17 @@ def _get_name(self):
 				unread=False
 			# Translators: when an email is unread
 			if unread: textList.append(_("unread"))
+			try:
+				mapiObject=selection.mapiObject
+			except COMError:
+				mapiObject=None
+			if mapiObject:
+				v=comtypes.automation.VARIANT()
+				res=NVDAHelper.localLib.nvdaInProcUtils_outlook_getMAPIProp(self.appModule.helperLocalBindingHandle,self.windowThreadID,mapiObject,PR_LAST_VERB_EXECUTED,ctypes.byref(v))

```

Comment:
Also, since this line is quite long, please split the arguments onto multiple lines.